app/src/app.ts

```diff
@@ -32,18 +38,45 @@ class App {
 
     scene.clearColor = new Color4(0, 0, 0, 0);
 
-    var box: Mesh = MeshBuilder.CreateBox("box", { size: 1 }, scene);
-    var material: StandardMaterial = new StandardMaterial("normal", scene);
-    material.diffuseTexture = new Texture("textures/amiga.jpg", scene);
+    // create players
+    this.players = Player.createPlayers(
```

Comment:
He you can initialize new objects with Player class constructor and then add them into players array.